pkgs/by-name/ri/ringboard/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libxkbcommon,
+  vulkan-loader,
+  stdenv,
+  wayland,
+  nix-update-script,
+  testers,
+  ringboard,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ringboard";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "SUPERCILEX";
+    repo = "clipboard-history";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Jk7PEXfdGKRICX7aC/3Ktkk4OWlT5IGON2Dp/AkUC7A=";
+  };
+
+  cargoHash = "sha256-I756jz2jwSXQuRM2el7VS0MCFXWzLCbEwX0HW7Rfnpg=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    libxkbcommon
+    vulkan-loader
+  ] ++ lib.optionals stdenv.isLinux [ wayland ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests = testers.testVersion { package = ringboard; };
+  };
+
+  meta = {
+    description = "a clipboard manager for Linux";
```

According to the [recommendations for meta attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) description should
* not start with an article
* start with a capital letter
```suggestion
    description = "Clipboard manager for Linux";
```